packages/blade/src/components/Button/BaseButton/buttonTokens.ts

```diff
@@ -0,0 +1,50 @@
+import type { ThemeTokens } from '../../../tokens/theme/theme.d';
+import type { TypographyPlatforms } from '../../../tokens/global/typography';
+
+export type ButtonMinHeight = '48px' | '40px' | '32px' | '28px';
+
+export type ButtonTypography = {
+  [key in TypographyPlatforms]: {
+    fonts: {
+      size: Record<string, keyof ThemeTokens['typography'][key]['fonts']['size']>;
```

Comment:
`Record<string, keyof ThemeTokens['typography'][key]['fonts']['size']>;` in this instead of `string` it should be union of allowed sizes right? 

Basically 👇 ?

`Record<'xsmall' | 'small' | 'medium' | 'larger', keyof ThemeTokens['typography'][key]['fonts']['size']>;`

Reply:
Yup, makes more sense